app/src/main/java/com/ucmobiledevelopment/freeways/service/IncidentService.kt

```diff
@@ -22,33 +22,31 @@ class IncidentService : IIncidentService {
     override suspend fun fetchIncidents(fromCaseYear: Int, toCaseYear: Int, state: Int, county: Int) : List<Incident>? {
 
         return withContext(Dispatchers.IO) {
-            var incidentList: MutableList<Incident> = mutableListOf()
```

Comment:
changed var to val as the value never changes.

Reply:
This duplicates a previous review. When that occurs, first in wins.